gravitee-apim-cypress/cypress/model/plan.ts

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export enum ValidationMode {
+  AUTO = 'AUTO',
+  MANUAL = 'MANUAL',
+}
+
+export enum SecurityType {
+  KEY_LESS = 'KEY_LESS',
+  API_KEY = 'API_KEY',
+  OAUTH2 = 'OAUTH2',
+  JWT = 'JWT',
+}
+
+export enum TypeType {
+  API = 'API',
+  CATALOG = 'CATALOG',
+}
+
+export enum StatusState {
+  STAGING = 'STAGING',
+  PUBLISHED = 'PUBLISHED',
+  CLOSED = 'CLOSED',
+  DEPRECATED = 'DEPRECATED',
+}
+
+export enum MethodType {
+  CONNECT = 'CONNECT',
+  DELETE = 'DELETE',
+  GET = 'GET',
+  HEAD = 'HEAD',
+  OPTIONS = 'OPTIONS',
+  PATCH = 'PATCH',
+  POST = 'POST',
+  PUT = 'PUT',
+  TRACE = 'TRACE',
+  OTHER = 'OTHER',
+}
+
+export interface Policy {
```

Comment:
policy.ts ?

Reply:
Not sure anymore why I've added that. So, I remove that until it's needed.